src/Cache/MomentoStore.php

```diff
@@ -0,0 +1,115 @@
+<?php
+
+namespace Momento\Cache;
+
+use Illuminate\Cache\TaggableStore;
+use Illuminate\Cache\TagSet;
+use Momento\Auth\EnvMomentoTokenProvider;
+use Momento\Cache\Errors\UnknownError;
+use Momento\Cache\SimpleCacheClient;
+
+class MomentoStore extends TaggableStore
+{
+    protected SimpleCacheClient $client;
+    protected string $cacheName;
+
+    public function __construct(string $cacheName, int $defaultTtl)
+    {
+        $authProvider = new EnvMomentoTokenProvider('MOMENTO_AUTH_TOKEN');
+        $this->client = new SimpleCacheClient($authProvider, $defaultTtl);
+        $this->cacheName = $cacheName;
+        $this->client->createCache($cacheName);
+    }
+
+    public function get($key)
+    {
+        $result = $this->client->get($this->cacheName, $key);
+        if ($result->asHit()) {
+            return $result->asHit()->value();
+        } elseif ($result->asMiss()) {
+            return null;
+        }
+    }
+
+    public function many(array $keys)
+    {
+        throw new UnknownError("many operations is currently not supported.");
+    }
+
+    public function put($key, $value, $seconds)
+    {
+        $result = $this->client->set($this->cacheName, $key, $value, $seconds);
+        if ($result->asSuccess()) {
+            return true;
+        } else {
+            return false;
+        }
+    }
+
+    public function putMany(array $values, $seconds)
+    {
+        throw new UnknownError("putMany operations is currently not supported.");
+    }
+
+    public function increment($key, $value = 1)
+    {
+        return false;
```

Comment:
Why is this one returning false instead of erroring like the others?

Also, I wonder if we should go ahead and do a client-side implementation for `putMany`, `increment`, and `decrement` for now.  We might want to have a zoom to discuss this.

Reply:
It looks like `increment` is used when `RouteServiceProvider` is booting up in the weather app. So an error is thrown, I was not able to start the app.

Reply:
okay then in that case let's do a client side implementation of this, where you read the key, if it doesn't exist set it to 0, then increment it, then save it.